src/settingsdialoggeneral.cpp

```diff
@@ -15,6 +15,10 @@ GeneralSettingsTab::GeneralSettingsTab(Settings& s, SettingsDialog& d)
   addLanguages();
   selectLanguage();
 
+  // Set scrollarea to transparent
+  ui->generalScrollArea->setStyleSheet("QScrollArea { background-color: transparent; }");
+  ui->generalScrollAreaWidgetContents->setStyleSheet("background-color: transparent;");
```

Comment:
The stylesheet can be set in the `.ui`, maybe it'd be better?

Reply:
Tried it.  Didn't work for some reason. /shrug.

Reply:
Works fine here... I added this to `generalScrollArea` stylesheet:

```
#generalScrollArea { background-color: transparent; }
#generalScrollAreaWidgetContents { background-color: transparent; }
```

Reply:
Did the thing